clearly-defined/src/main/kotlin/ClearlyDefinedService.kt

```diff
@@ -239,15 +239,41 @@ interface ClearlyDefinedService {
     )
 
     /**
-     * See https://github.com/clearlydefined/service/blob/b339cb7/schemas/curations-1.0.json#L64-L83.
+     * See https://github.com/clearlydefined/service/blob/b339cb7/schemas/curations-1.0.json#L64-L83 and
+     * https://docs.clearlydefined.io/using-data#a-note-on-definition-coordinates.
      */
     @JsonInclude(JsonInclude.Include.NON_NULL)
     data class Coordinates(
-        val name: String,
-        val namespace: String? = null,
+        /**
+         * The type of the component. For example, npm, git, nuget, maven, etc. This talks about the shape of the
```

Comment:
Maybe "describes" or "defines" instead of "talks about"?

Reply:
I'm also not super-fond of this wording, but I literally copied it from https://docs.clearlydefined.io/using-data#a-note-on-definition-coordinates and wanted to keep it like this, except some trivial changes to make it grammatically correct in our context.